server/routes/unleashRoutes.ts

```diff
@@ -84,22 +84,15 @@ export const unleashToggles = (toggles: any, valgtEnhet: any, userId: any) => {
         user: userId,
       }
     ),
-    "syfo.behandlerdialog.legeerklaring": unleash.isEnabled(
-      "syfo.behandlerdialog.legeerklaring",
-      {
-        valgtEnhet: valgtEnhet,
-        user: userId,
-      }
-    ),
-    "syfo.behandlerdialog.returlegeerklaring": unleash.isEnabled(
-      "syfo.behandlerdialog.returlegeerklaring",
+    "syfo.motebehov.tilbakemelding": unleash.isEnabled(
```

Comment:
Legger til *to* nye toggles?

Reply:
Jeg tror det er git-diffen som blir litt rar her, siden jeg fjernet to stk, og la til den nye nederst. Så den `tilbakemelding` fantes fra før også, hvis du ser på forrige versjon av filen 😄